commands/cli.go

```diff
@@ -29,7 +35,7 @@ func CreateCli(version string) *cli.App {
 func awsNuke(c *cli.Context) error {
 	logging.Logger.Infoln("Retrieving all active AWS resources")
 
-	account, err := aws.GetAllResources()
+	account, err := aws.GetAllResources(c.StringSlice("exclude"))
```

Comment:
You should validate the list of regions passed in to ensure each one corresponds to a valid AWS region. We don't want to blow away a region because someone made a typo (e.g., us-wes-1).